src/main/java/dino/Dino.java

```diff
@@ -0,0 +1,81 @@
+package dino;
+
+import dino.command.*;
+import dino.exception.*;
```

Comment:
Would it be better if your imported classes are explicitly stated for better documentation? This would also comply with the coding standards in the CS2103T website as well 😄